src/libs/fileDownload/getAttachmentDetails.ts

```diff
@@ -3,10 +3,8 @@ import tryResolveUrlFromApiRoot from '../tryResolveUrlFromApiRoot';
 
 /**
  * Extract the thumbnail URL, source URL and the original filename from the HTML.
- * @param {String} html
- * @returns {Object}
  */
-export default function getAttachmentDetails(html) {
+export default function getAttachmentDetails(html: string): unknown {
```

Comment:
Why `unknown` if you explicitly return an object  wtih `previewSourceURL`, `sourceURL` and `originalFileName`?